.github/ISSUE_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,17 @@
+<!-- Please provide all the information below.  -->
+
+### Issue describing the changes in this PR
+
+resolves #issue_for_this_pr
+
+### Pull request checklist
+
+* [ ] My changes **do not** require documentation changes
+    * [ ] Otherwise: Documentation issue linked to PR
+* [ ] My changes **should not** be added to the release notes for the next release
+    * [ ] Otherwise: I've added my notes to `release_notes.md`
+
```

Comment:
Reminder on fix needs to be ported to V2?

Reply:
Reminder on UnitTests and E2E tests either automated or manual.

Reply:
Great call out on the V2 backport.

Debated on the tests, but I think it would be good to add.